mods/d2k/weapons/smallguns.yaml

```diff
@@ -81,7 +81,6 @@ HMG:
 	Report: 20MMGUN1.WAV
 	Warhead@1Dam: SpreadDamage
 		Spread: 160
```

Comment:
Is this intentional? It's weird for the Trike/Raider to have only 1/3 the default Spread.

Reply:
I guess I must have missed that; good catch.

Reply:
Yes, I checked and the `HMG` should have a spread of 480, which gets inherited from `^MG`. Removed the override.